### DISCUSSION TITLE
Card Graphics not as intended

### DISCUSSION THREAD

**Discussion by Boabee** [UI-Lovelace-Minimalist/UI#178] [Q&A] [answered]
Hi everyone, this is my first discussion and Im new to everything, go easy please.
I have installed the theme by Yaml. I have got my dashboard pages up and running via the tutorial and my first 'chip' is displayed with weather/temp etc. The graphic has its coloured weather logo with its rounded edges, as intended (i presume as per button_card_template.yaml).

I have now began adding more buttons (light slider and person card) in a horizontal stack but the graphics are not appearing in the theme format and they are also not working as intended to on the templates page of the Minimalist Wiki. I have them appearing in the dashboard but for example, the light slider does not slide and the edges are square and the person card does not have a blue icon.

Can anyone shed any light on this subject? Thanks

Note: I made the background black for edge reference.
![example](https://user-images.githubusercontent.com/52744496/147743060-f65fee33-0366-4b2a-99bf-8dfd61574176.png)

**Comment by sildehoop** [comment, marked as answer]
Have you selected the theme in your user preferences??

**Reply by Boabee** [reply]
You my friend are my saviour! I missed that part! Thank you!!
